chronicles-rulebook.md

```diff
@@ -2735,7 +2725,7 @@ Item production is the process by which characters are able to craft consumable
 #### Rank III
 
 ##### Repair Lock
-   >The tinkerer is able to fix a previously broken lock. The repair of the lock takes 1 minute of  time in addition to the three ranks of raw materials.
+   >The tinkerer is able to fix a previously broken lock. The repair of the lock takes 1 minute of  time in addition to the three ranks of raw materials. This is not a tagged item.
```

Comment:
Extra space between "of" and "time" here.

Side consideration:  If it is not a tagged item, this is a Passive ability for a Tinkerer.  Ever thought about similar for the others?